src/app/shared/pipes/digit-group-separator/digit-group-separator.pipe.ts

```diff
@@ -0,0 +1,14 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'digitGroupSeparator'
+})
+export class DigitGroupSeparatorPipe implements PipeTransform {
+  transform(value: number | undefined): string {
+    if (value !== undefined && value !== null) {
+      return value.toLocaleString();
+    } else {
+      return '';
+    }
+  }
```

Comment:
```suggestion
  transform(value: number | undefined): string | undefined {
    value ? value.toLocaleString() : value
  }
```

Reply:
Don't know if this is something we need to worry about but `toLocaleString()` on a string does not format the string to be a comma delimited number. Just something to keep in mind if we want to format a field that is a string in the data but a number on the screen